convert/poe2arb/converter.go

```diff
@@ -57,6 +58,15 @@ func (c *Converter) Convert(output io.Writer) error {
 	prefixedRegexp := regexp.MustCompile("(?:([a-zA-Z]+):)?(.*)")
 	var errs []error
 
+	// Sort terms by key alphabetically
+	sort.SliceStable(jsonContents, func(i, j int) bool {
```

Comment:
Theres a polymorphic version of this function in `slices` https://pkg.go.dev/slices#SortStableFunc